### PR TITLE
Fix F8/BF8 failed cases for GWVW=8 and Beta != 0 

### PR DESCRIPTION
**Pull request by geotseng-amd**
In this PR, we fixed the **fp8** and **bf8** failed cases in large vector widths.
1. Ticket: https://ontrack-internal.amd.com/browse/SWDEV-494325
2. Fixed in /hipBLASLt/tensilelite/Tensile/Components/GlobalWriteBatch.py

### CODE REVIEW

**Comment by geotseng-amd** [comment]
Local test results for PR#1315 are all passed.

[33m=========== [32m78 passed[0m, [33m[1m32 skipped[0m, [33m[1m2 warnings[0m[33m in 9251.44s (2:34:11)[0m[33m ============[0m
py3: exit 0 (9251.70 seconds) /workspace/fork/hipBLASLt/tensilelite> py.test -v --basetemp=/tmp/.tensile-tox/py3/tmp --junit-xml=/workspace/fork/hipBLASLt/tensilelite/python_tests.xml --junit-prefix=py3 --color=yes -n 4 --prebuilt-client=/tmp/.tensile-tox/py3/client/0_Build/client/tensile_client Tensile/Tests -m common pid=269783
  py3: OK (9401.40=setup[12.09]+cmd[0.69,136.92,9251.70] seconds)
  congratulations :) (9401.47 seconds)

**Review by jichangjichang — COMMENTED** [review]
add beta =1  in gtest test item "matmul_heuristic_all_solutions_real_1byte"

**Comment by geotseng-amd** [comment]
Done 
CI: "matmul_heuristic_all_solutions_real_1byte" passed.


[----------] Global test environment tear-down
[==========] 640 tests from 1 test suite ran. (57923 ms total)
[  PASSED  ] 640 tests.
hipBLASLt version: 1100

command line: ./build/release/clients/staging/hipblaslt-test --gtest_filter=*matmul_heuristic_all_solutions_real_1byte* --gtest_output=xml --gtest_color=yes --gtest_repeat=1

**Comment by jichangjichang** [comment]
> Done CI: "matmul_heuristic_all_solutions_real_1byte" passed.
> 
> [----------] Global test environment tear-down [==========] 640 tests from 1 test suite ran. (57923 ms total) [ PASSED ] 640 tests. hipBLASLt version: 1100
> 
> command line: ./build/release/clients/staging/hipblaslt-test --gtest_filter=_matmul_heuristic_all_solutions_real_1byte_ --gtest_output=xml --gtest_color=yes --gtest_repeat=1

CI is broken. You should run full Gtest but not just new added test.

**Comment by geotseng-amd** [comment]
All local CI test results for PR#1315 are passed.

[0;32m[----------] [mGlobal test environment tear-down
[0;32m[==========] [m48758 tests from 13 test suites ran. (805817 ms total)
[0;32m[  PASSED  ] [m48758 tests.
hipBLASLt version: 1100

command line: ./build/release/clients/staging/hipblaslt-test --gtest_output=xml --gtest_color=yes --gtest_repeat=1 

100% tests passed, 0 tests failed out of 38

Total Test time (real) = 1025.29 sec

**Review by jichangjichang — DISMISSED** [review]

File: tensilelite/Tensile/Components/GlobalWriteBatch.py [hunk 1]
```diff
@@ -2008,7 +2008,8 @@ def _addSumAlphaWithCBeta(self, kernel, ss, gwvw, elementIdx, vc0, tmpVgpr, cvtV
               sb = 0 if vi ==0 else 1
               module.add(VCvtPkFP8toF32(dst=vgpr(tmpVgpr, 2), src=vgpr(dataV), vop3=VOP3PModifiers(op_sel=[sb])))
             else:
-              sb = SelectBit.WORD_0 if vi == 0 else SelectBit.WORD_1
+              # Enable WORD_0 of 2-nd VGPR with vi=4 for vw=8
+              sb = SelectBit.WORD_0 if vi%4 == 0 else SelectBit.WORD_1
```
Comment (nakajee) [inline tensilelite/Tensile/Components/GlobalWriteBatch.py:2013]: Don't we need to apply the same logic for NoSDWA case at line 2009?

Comment (geotseng-amd) [reply tensilelite/Tensile/Components/GlobalWriteBatch.py:2013]: Sure. Done, Please approve it.

Comment (jichangjichang) [reply tensilelite/Tensile/Components/GlobalWriteBatch.py:2013]: Please help to remove #cm review

**Review by Serge45 — DISMISSED** [review]

**Review by jichangjichang — APPROVED** [review]

**Review by Serge45 — APPROVED** [review]

File: tensilelite/Tensile/Components/GlobalWriteBatch.py (hunk 1 above)
Comment (geotseng-amd) [reply tensilelite/Tensile/Components/GlobalWriteBatch.py:2013]: Done

**Pull request merged**